tmk_core/common/keycode.h

```diff
@@ -481,9 +484,9 @@ enum internal_special_keycodes {
     KC_MS_WH_LEFT,
     KC_MS_WH_RIGHT,     /* 0xFC */
     /* Mousekey accel */
-    KC_MS_ACCEL0,
-    KC_MS_ACCEL1,
-    KC_MS_ACCEL2        /* 0xFF */
+    KC_MS_ACCEL3,
+    KC_MS_DNSPED,
+    KC_MS_UPSPED        /* 0xFF */
```

Comment:
Had to remove the existing MS_ACCEL because the last register 0xFF was already taken up